### DISCUSSION TITLE
How to get the edit icon?

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#880] [Q&A] [answered]
Hi team,

I am able to find the edit icon on Paste website but not in the sketch library.
How can I get the edit icon? Am I looking at the right sketch library file?

<img width="772" alt="Screen Shot 2020-10-22 at 5 08 00 PM" src="https://user-images.githubusercontent.com/56697620/96939092-45429a80-1489-11eb-8cac-f762e196179c.png">

<img width="637" alt="Screen Shot 2020-10-22 at 5 07 48 PM" src="https://user-images.githubusercontent.com/56697620/96939100-4a9fe500-1489-11eb-80f1-6e60ead3f240.png">

**Comment by serifluous** [comment, marked as answer]
Hey @charleneyjh1! We recently changed our design release process so new design libraries come out every 6 weeks to make  sure we're testing new components reliably.

Unfortunately, this means new things (like the edit icon) might still be in a design branch, while they're out in code. You can find the icon in our November release branch here: https://share.goabstract.com/927bc184-b0f0-4d28-871e-b417be9e118f?sha=0416cf2f96b62c94e6c1c159271c2b28fa4fad30

For now, I'd recommend using a placeholder icon and swapping to the EditIcon when it's available in a few weeks (we'll follow up here when it's out). If you _absolutely_ need the pencil in your mockups, you can copy and paste it out of the file linked above.
